run-tests

```diff
@@ -7,14 +7,14 @@ fi
 if pkg-config vchan-socket; then
     export CFLAGS="--coverage -DCOVERAGE"
     export LDFLAGS=--coverage
-    make -C libqrexec BACKEND_VMM=socket
-    make -C agent BACKEND_VMM=socket
-    make -C daemon BACKEND_VMM=socket
+    make -C libqrexec BACKEND_VMM=socket clean all
+    make -C agent BACKEND_VMM=socket clean all
+    make -C daemon BACKEND_VMM=socket clean all
 else
     echo "libvchan-socket not available, skipping socket tests"
     export SKIP_SOCKET_TESTS=1
 fi
 
 set -x
 
-python3 -m coverage run -m pytest -o python_files=*.py -v qrexec/tests "$@"
+python3 -m coverage run -m pytest -o python_files=*.py -v "$@"
```

Comment:
If you change this, add the argument you removed to the call in `.gitlab-ci.yml`, otherwise it collects unintended files.